.circleci/config.yml

```diff
@@ -157,6 +158,21 @@ jobs:
            _build/debug/velox/examples/velox_example_expression_eval
            _build/debug/velox/examples/velox_example_opaque_type
 
+  linux-benchmarks:
+    executor: build
+    steps:
+      - checkout
+      - run:
+          name: "Building and running baseline (main)"
+          command: |
+            #git checkout HEAD^1 
+            make benchmarks-baseline
+      - run:
+          name: "Building and running revision"
+          command: |
+            #git checkout -
```

Comment:
Here we would checkout back the current revision, then run benchmarks against comparing to the dumps we just took.

Reply:
This is a great start . One question is How does this work the first time someone adds a benchmark ?

Reply:
For the first time, I guess that when we check out main and run the dumps, no dump will be generated for the new benchmark, so we'll likely skip it when running the comparison.